samples/command/consumer/src/main.rs

```diff
@@ -13,24 +13,39 @@ use samples_common::utils::{discover_digital_twin_provider_using_ibeji, retrieve
 use samples_protobuf_data_access::sample_grpc::v1::digital_twin_consumer::digital_twin_consumer_server::DigitalTwinConsumerServer;
 use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::digital_twin_provider_client::DigitalTwinProviderClient;
 use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::InvokeRequest;
+use serde_derive::{Deserialize, Serialize};
 use std::net::SocketAddr;
 use tokio::time::{sleep, Duration};
 use tonic::transport::Server;
 use uuid::Uuid;
 
+#[derive(Debug, Serialize, Deserialize)]
+struct ShowNotificationRequestPayload {
+    #[serde(rename = "Notification")]
+    notification: sdv::hmi::show_notification::request::TYPE,
+    #[serde(rename = "$metadata")]
+    metadata: Metadata,
+}
+
 /// Start the show notification repeater.
 ///
 /// # Arguments
 /// `provider_uri` - The provider_uri.
 /// `consumer_uri` - The consumer_uri.
 fn start_show_notification_repeater(provider_uri: String, consumer_uri: String) {
     debug!("Starting the Consumer's show notification repeater.");
+
+    let request_payload: ShowNotificationRequestPayload = ShowNotificationRequestPayload {
```

Comment:
the type annotation here is not necessary

Reply:
I prefer to make it explicit.